addons/website_slides/controllers/mail.py

```diff
@@ -25,9 +25,9 @@ def portal_chatter_post(self, res_model, res_id, message, **kw):
                 # apply karma gain rule only once
                 request.env.user.add_karma(slide_channel.karma_gen_channel_rank)
             result.update({
-                'rating_value': rating_value,
+                'default_rating_value': rating_value,
                 'ratingAvg': slide_channel.rating_avg,
-                'rating_count': slide_channel.rating_count
+                'ratingTotal': slide_channel.rating_count
```

Comment:
Please don't mix and match camelCase and snake_case. Since existing RPC methods (like search_reach) return snake_case, please stick to that convention.